### PR TITLE
Feature/query multiple tables

### PR DESCRIPTION
**Pull request by Fourteen98**
### Questions?

- What animals belong to Melody Pond? 
image

- List of all animals that are pokemon (their type is Pokemon).
image

- List all owners and their animals, remember to include those that don't own any animal.
image

- How many animals are there per species?
image

- List of all Digimon owned by Jennifer Orwell.
image

- List all animals owned by Dean Winchester that haven't tried to escape.
image

-  Who owns the most animals?
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -69,3 +69,20 @@ SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
 
 
+/* What animals belong to Melody Pond? */
+SELECT name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+
+/* List of all animals that are pokemon (their type is Pokemon). */
+SELECT animals.name FROM animals JOIN species ON animals.species_id  = species.id WHERE species.name = 'Pokemon';
+
```
Comment (brytebee) [inline queries.sql:77]: - [x] **Required:** You are missing a required query for all names in the owners' table. Those that don't own any animals are inclusive.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Fourteen98👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design.✔️
- The Readme looks professional.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Fourteen98) [reply queries.sql:77]: oh okay, LEFTJOIN. thank you . will make changes now

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Fourteen98  🔥 

Good job on implementing the required changes. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**